docs/guide/search/solr.md

```diff
@@ -22,12 +22,11 @@ The example presents a configuration with a single core. For configuring Solr in
 
 !!! note "Solr versions"
 
-    `ezplatform-solr-search-engine` v1 still supports Solr 6.
-    Since v2 `ezplatform-solr-search-engine` works only with Solr 7.
+    For v3 `ezplatform-solr-search-engine` works with Solr 7 and 8, using an supported LTS versions of Solr is recommended.
```

Comment:
```suggestion
    v3 of `ezplatform-solr-search-engine` works with Solr 7 and Solr 8.
    It is recommended that you use a current major version of Solr.
```

Lucene does not refer to its versions as "LTS". They use terms like major, previous major, or EOL. Alternatively, we could say "It is recommended that you do not use an EOL version of Solr."

Reply:
The thing behind the recommendation:
1. 7.7 is given bug fixes still, while 8.x receives new features and sometimes possible breaks in minor releases
2. Platform.sh [supports 7.7, and 8.6](https://docs.platform.sh/configuration/services/solr.html), however they probably will take some time to add 8.7 and 8.8 when it arrives, so we try to avoid this kind of friction.

But maybe we should rather be specific to recommend versions:
- We test on
- available on p.sh

That would mean either 7.7 or 8.6 right now, and then we update it once 8.8 is out and available on psh.

Reply:
"That would mean 7.7 or 8.6 right now and then we update it once 8.8 is out and available on psh." - makes perfect sense to me

Reply:
```suggestion
    For v3 `ezplatform-solr-search-engine` works with Solr 7 and 8, using 7.7 or 8.6 of Solr is currently recommended.
```